XMonad/Layout/IndependentScreens.hs

```diff
@@ -134,8 +160,7 @@ countScreens = fmap genericLength . liftIO $ openDisplay "" >>= liftA2 (<*) getS
 -- > logHook = let log screen handle = dynamicLogWithPP . marshallPP screen . pp $ handle
 -- >           in log 0 hLeft >> log 1 hRight
 marshallPP :: ScreenId -> PP -> PP
-marshallPP s pp = pp { ppRename = ppRename pp . unmarshallW
-                     , ppSort = fmap (marshallSort s) (ppSort pp) }
+marshallPP s pp = pp { ppRename = ppRename pp . unmarshallW }
```

Comment:
We still need something like `ppSort = ppSort pp . workspacesOn s` here.

Reply:
Thanks for pointing out, I have added it now.

Reply:
Now the docs are out of date again. And the marshallPP doc needs to be adjusted, too, as it's no longer accurate.

Reply:
I have updated the docs. Does it look OK now?